curriculum/challenges/english/10-coding-interview-prep/the-odin-project/understand-ordered-and-unordered-list-question-a.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 637f4e3e72c65bc8e73dfe24
+videoId: kcHKFZBVtf4
+title: Understand Ordered and Unordered List Question A
```

Comment:
This will require changes in other places, but I think a better title might be "Understand Ordered and Unordered Lists..."

We could even go with "Ordered and Unordered Lists...", or even just "Lists..." like on The Odin Project.